<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyScreenSaver</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip{
	color : black;
}
QWidget#centralWidget{
	background : #333333;
}
QDialog{
	background : #333333;
}
QLCDNumber{
background : white;
}

QTabWidget{
border: none;
}
QTabBar::tab{
background : black;
color : white;
}
QTabBar::tab::selected{
background : #808080;

}
QGroupBox{
	color : white;
	margin-top : 2ex;
	border-radius : 5px;
	padding : 8px;
	padding-top : 13px;
	 border-style: outset;
 	 border-width: 2px;
	 border-color: #1f1f1f;
}
QMenuBar#menuBar{

	background : grey;
	color : white;
	 border-bottom-style: outset;
 	 border-bottom-width: 1px;
	 border-bottom-color: #222222;
}
QMenu{
	background : grey;
	color : #1f1f1f;
}
QMenu::item:selected{
	background : #c9c9c9;
}
QStatusBar#statusBar{
	background : grey;
	color : white;
	border-top-style: outset;
 	border-top-width: 1px;
	border-top-color: #222222;
}
QPushButton{
	color : white;
	background : grey;
	padding : 8px 5px;
	border-radius : 3px;
	min-width : 60px;
}
QPushButton:hover{
	background : #c9c9c9;
}

QPushButton:pressed{
	background : #474747;
}

QPushButton:disabled{
	color : #474747;
}
QLineEdit{
	color : white;
	background : #111111;
	padding : 3px;
	border-radius : 3px;
	border-color : #474747;
	border-width : 1px;
	border-style : inset;
}
QTextEdit{
	color : white;
	background : #111111;
	padding : 3px;
	border-radius : 3px;
	border-color : white;
	border-width : 1px;
}
QComboBox{
	color : white;
	background : #111111;
	padding : 3px;
	min-width : 40px;
	border-radius : 3px;
	border-style : inset;
	border-width : 1px;
	border-color : #474747;
}
QComboBox::drop-down{
	
}
QComboBox QAbstractItemView{
	background-color : black;
	color : white;
}
QSlider::handle{
	background : grey;
}
QSlider::handle:hover{
	background : #c9c9c9;
}
QSlider::handle:pressed{
	background : #474747;
}
QLabel{
	color : white;
}
QRadioButton{
	color : white;
}
QCheckBox{
	color : white;
}
QRadioButton::indicator{
	 border-style: outset;
 	 border-width: 2px;
	 border-color: #1f1f1f;
	background : white;
	border-radius : 6px;
}
QRadioButton::indicator:checked{
	 border-style: outset;
 	 border-width: 2px;
	 border-color: white;
	background : #1f1f1f;
	border-radius : 6px;
}
QCheckBox::indicator{
	 border-style: outset;
 	 border-width: 2px;
	 border-color: #1f1f1f;
	background : white;
}
QCheckBox::indicator:checked{
	 border-style: outset;
 	 border-width: 2px;
	 border-color: white;
	background : #1f1f1f;
}
QHeaderView::section{
	border-style : outset;
	border-width : 1px;
	border-color : #c9c9c9;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Configuration:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Choose an application:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="appPathEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browseBtn">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Idle time(mins):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="idleTimeEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="turnOnUponLaunchCheck">
           <property name="text">
            <string>Turn on upon launch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="runOnWindowStartupCheck">
           <property name="text">
            <string>Run on windows startup</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="runAsAdminCheck">
         <property name="text">
          <string>Run as administrator</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="easyqrc.qrc">
          <normaloff>:/assets/toggle-mid.png</normaloff>:/assets/toggle-mid.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyBtn">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="easyqrc.qrc"/>
 </resources>
 <connections/>
</ui>
